### PR TITLE
Updated Embedding Setup to Fix Metaclass Error

### PR DESCRIPTION
**Pull request by genaimagician**
Please refer to this open issue for more details for the error I fixed:
https://github.com/GoogleCloudPlatform/applied-ai-engineering-samples/issues/80

### CODE REVIEW

File: genai-on-vertex-ai/developer_productivity_with_genai/5_codey_talk_to_codebase_example.ipynb [hunk 1]
```diff
@@ -1,3308 +1,3168 @@
 {
```
Comment (RajeshThallam) [inline genai-on-vertex-ai/developer_productivity_with_genai/5_codey_talk_to_codebase_example.ipynb:1]: <p>Can you please update the year in copyright? Also, Last updated date.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (RajeshThallam) [inline genai-on-vertex-ai/developer_productivity_with_genai/5_codey_talk_to_codebase_example.ipynb:1]: <p>Would it make sense to update to latest version of Vertex AI SDK and langchain?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (RajeshThallam) [inline genai-on-vertex-ai/developer_productivity_with_genai/5_codey_talk_to_codebase_example.ipynb:1]: <p>Clear the output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (RajeshThallam) [inline genai-on-vertex-ai/developer_productivity_with_genai/5_codey_talk_to_codebase_example.ipynb:1]: <p>Clear the output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Comment (RajeshThallam) [inline genai-on-vertex-ai/developer_productivity_with_genai/5_codey_talk_to_codebase_example.ipynb:1]: <p>Clear the output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Comment (RajeshThallam) [inline genai-on-vertex-ai/developer_productivity_with_genai/5_codey_talk_to_codebase_example.ipynb:1]: <p>Clear the output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>

Comment (RajeshThallam) [inline genai-on-vertex-ai/developer_productivity_with_genai/5_codey_talk_to_codebase_example.ipynb:1]: <p>Clear the output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

Comment (genaimagician) [reply genai-on-vertex-ai/developer_productivity_with_genai/5_codey_talk_to_codebase_example.ipynb:1]: Done

Comment (genaimagician) [reply genai-on-vertex-ai/developer_productivity_with_genai/5_codey_talk_to_codebase_example.ipynb:1]: Done

Comment (genaimagician) [reply genai-on-vertex-ai/developer_productivity_with_genai/5_codey_talk_to_codebase_example.ipynb:1]: Done

Comment (genaimagician) [reply genai-on-vertex-ai/developer_productivity_with_genai/5_codey_talk_to_codebase_example.ipynb:1]: Done

Comment (genaimagician) [reply genai-on-vertex-ai/developer_productivity_with_genai/5_codey_talk_to_codebase_example.ipynb:1]: Done

Comment (genaimagician) [reply genai-on-vertex-ai/developer_productivity_with_genai/5_codey_talk_to_codebase_example.ipynb:1]: Done.

Comment (genaimagician) [reply genai-on-vertex-ai/developer_productivity_with_genai/5_codey_talk_to_codebase_example.ipynb:1]: I just updated them to the versions that were fully tested. google-cloud-aiplatform==1.47.0 langchain==0.1.14 langchain-google-vertexai==0.1.3

The reason is that when I upgraded everything to the latest one, there were libraries conflicts.

**Review by RajeshThallam — APPROVED** [review]
LGTM

**Pull request merged**